angular/src/components/icon.component.ts

```diff
@@ -60,24 +53,24 @@ export class IconComponent implements OnChanges {
   }
 
   get iconCode(): string {
-    return IconMap[this.icon];
+    return BitwardenIconsService.getIconCode(this.icon);
   }
 
   protected load() {
     switch (this.cipher.type) {
       case CipherType.Login:
-        this.icon = "fa-globe";
+        this.icon = "bwi-globe";
```

Comment:
I was hoping we could do something like `BitwardenIcons.Globe`, and be certain it exists due to type safety. Will also make it easier to find future references since it will fail to compile.

Reply:
I've added a Tech Debt [task](https://app.asana.com/0/1201461753012676/1201602342451573) that will follow up on using typed icon references.